db/schema.rb

```diff
@@ -11,11 +11,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 20161206021535) do
```

Comment:
@mscoutermarsh what is the best way to merge the schema version? Simply use the new version id and delete the old? Or delete the schema file, pull it from master, run migrations and push that?